Create community growth & sustainability section

Nolski opened the pull request:
### Summary
This section provides a guide that allows readers to develop a fully evidenced "community roadmap" to help with the issues of both growing and sustaining a community. Open@RIT developed it as part of their work assisting academic open work projects. There may be some overlap with the existing personas section.


Fixes #785

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)

<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

* A section which describes a general method of how to create a community roadmap for a project.


### What should a reviewer concentrate their feedback on?

<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

- [ ] Confirm material in this section does not conflict with other sections
- [ ] Copy editing, typos, etc.
- [ ] Is further information needed for new contributors?

### Acknowledging contributors

<!-- Please select the correct box -->

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+# Open Source Community Building Guide
```
1. I would change the title of this section to something more specific, akin to your title, like "Community Roadmaps"
2. Don't forget to add the specific syntax at the top of your chapter, as outlined in our [Style guide](https://book.the-turing-way.org/community-handbook/style/style-crossref).
```suggestion
(pd-community-roadmap)=
# Open Source Community Building Guide
```

Nolski replied:
I went ahead and updated this in the latest commit, let me know if that looks to be correct and if I'm doing the subchapter syntax correct :)

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+* **Intrinsic Motivations:** Personal growth, learning opportunities, passion for the project.
+* **Extrinsic Motivations:** Recognition, monetary rewards, career advancement.
+
+For instance, a developer might be motivated by the need to add specific functionality for their use case. At the same time, a technical writer may be more incentivized to contribute better documentation to onboard their teammates..
```
```suggestion
For instance, a developer might be motivated by the need to add specific functionality for their use case. At the same time, a technical writer may be more incentivized to contribute better documentation to onboard their teammates.
```

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+
+### Example: Open@RIT Community Building Roadmap
+
+An example roadmap document template has been developed by Open@RIT and is [available on Google Slides](https://docs.google.com/presentation/d/1iYWGpm9MUNnQIsF9Smh4w8wgkA68JWi2NBqJxi28kmE/edit#slide=id.gf80e76d98d_0_225)
```
This could be a great case study to add: https://the-turing-way.netlify.app/community-handbook/contributing/contributing-templates

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+**2. Questionnaire Development:** Develop focused questions for each stage of the contributor pathway and identify issues faced in contribution. The questions may vary highly depending on the individual's background and relation to the project. It is important for those conducting interviews to be flexible in their approach towards contributor interviews to allow for the most useful findings. A s ...
+
+
+### Example Questions:
```
Love these sample questions!

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+
+This guide outlines a comprehensive methodology for defining project goals, understanding contributor motivations, and creating a clear pathway for engagement. Drawing on best practices from the Mozilla Open Leadership Training Series and The Turing Way, this guide aims to help project maintainers foster a vibrant and sustainable open-source community. Below you will find guidelines for developin ...
+
+
```
If this section is specifically about building a community roadmap (given the title and description), I would suggest writing a small description here about how the methodology above contributes to the creation of a roadmap more specifically. It looks great otherwise!

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+
+## Understanding the Contribution Pathway
+
+The contribution pathway outlines the stages through which individuals engage with the project, from first hearing about it to taking on leadership roles. Each stage has specific objectives and actions to facilitate smooth transitions.
```
Are there resources you can add, or citations to put here?

Nolski replied:
See below

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+Choose an appropriate format for conducting the interviews, such as one-on-one interviews for in-depth insights, focus groups for collective discussions, or online surveys for broader reach. Ensure accurate data collection by recording responses and taking detailed notes. This helps capture nuanced insights and direct quotes that can enrich the analysis.
+
+**Examples:**
+
```
Looks like there are some extra spaces here with this list.

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,239 @@
+
+## Creating the Community Building Roadmap
+
+With the findings from the interviews, synthesize the key insights and prioritize actionable recommendations by developing Community Roadmap. The roadmap should be informed by thorough research and data collection, including interviews with key stakeholders. Below is a detailed narrative on creating a community building roadmap, incorporating insights from the attached Open@RIT presentation templ ...
```
```suggestion
Using the findings from the interviews, synthesize the key insights and prioritize actionable recommendations by developing Community Roadmap. The roadmap should be informed by thorough research and data collection, including interviews with key stakeholders. Below is a detailed narrative on creating a community building roadmap, incorporating insights from the attached Open@RIT presentation template.
```

aleesteele commented:
Hi @Nolski - thanks so much for creating this draft, this looks great! I just made a few minor formatting suggestions. 

The Open @ RIT could also be a great case study to add to the guides as well - perhaps in a different pull request, but I think it could a very valuable resource that folks could engage in, too. 😄

aleesteele commented:
Adding breadcrumbs to #3785

Nolski commented:
Apologies, I'm not sure what happened but I think my covid ridden brain fog tried rebasing on top of main and now external commits have made their way into this PR :( I'll try and resolve this in a bit when my brain is working properly...

aleesteele commented:
Oh no, @Nolski! You can cherry pick the commit you'd like to, OR rebase your branch to the main branch, and that should solve the commit issues! 🦾

Nolski commented:
Hi @aleesteele ! Sorry about taking so long, went ahead and rebased. This PR should be ready for review now :)

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,243 @@
+
+This guide outlines a comprehensive methodology for developing a Community Roadmap. This roadmap should serve as the central document that will allow maintainers to strategically understand how to acheive their goals related to creating a community of useful contributors necessary to maintain a healthy project.
+
+This roadmap will often have sections such as project goals, understanding contributor motivations, and creating a clear pathway for engagement. Drawing on best practices from the Mozilla Open Leadership Training Series and The Turing Way, this guide aims to help project maintainers foster a vibrant and sustainable open-source community. Below you will find guidelines for developing a strategy to ...
```
```suggestion
This roadmap will often have sections such as project goals, understanding contributor motivations, and creating a clear pathway for engagement. Drawing on best practices from the Mozilla Open Leadership Training Series and _The Turing Way_, this guide aims to help project maintainers foster a vibrant and sustainable open-source community. Below you will find guidelines for developing a strategy to growing your community as well as specific instructions for gathering the evidence needed to identify specific roadblocks to contribution.
```

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,243 @@
+
+### 1. Contributor Personas:
+
+* **Types of Contributors:** Developers, documenters, designers, community managers, etc.
```
Just adding a few more persona types!
```suggestion
* **Types of Contributors:** Developers, documenters, designers, community managers, project managers, researchers, etc.
```

aleesteele commented on book/website/project-design/community-growth-sustainability.md:
```diff
@@ -0,0 +1,243 @@
+* **Community Events:** Organize virtual meetups and webinars to engage with the community.
+
+**Long-Term Strategies:** Plan for sustainable growth and development of the community over time. Examples include:
+
```
There's a long line-break here for some reason!

aleesteele commented:
Added a few comments - I'll add this to our ask away channel for a few more reviews! Great work, @Nolski 😄 - thanks for adding these to the guides!

Nolski commented:
@aleesteele Sorry this is a very stale PR but if it is still welcome, I addressed your comments. I noticed that the `_toc.yml` is no longer used anymore. Do I need to add my page to some website config file for it to show up on the nav?
